### PR TITLE
Update game over message

### PR DESCRIPTION
**Pull request by VedangiPatel**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (m227chan) [inline index.html:51]: HI VEDANGI

Comment (vyomeshiyengar) [reply index.html:51]: very cool code

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
```
Comment (vyomeshiyengar) [inline index.html:48]: REVIEW!!

**Review by vyomeshiyengar — APPROVED** [review]

**Review by VedangiPatel — COMMENTED** [review]
looks good

File: index.html (hunk 1 above)
Comment (vyomeshiyengar) [inline index.html:51]:
```suggestion
<h2 hidden>Game over, refresh to play again tomorrow🧑‍💻 🤖!</h2>
```

**Review by vyomeshiyengar — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (VedangiPatel) [inline index.html:48]:
```suggestion
</table>
something
```

**Pull request merged**